src/components/ui/input.tsx

```diff
@@ -0,0 +1,25 @@
+import * as React from "react"
```

Comment:
Use the Input component for consistency around the website

Reply:
Which?
From shadcn?

Reply:
@Justfemi 
Someone's created the input component in a PR. I've pulled from upstream dev and merged already